final_project.py

```diff
@@ -0,0 +1,369 @@
+Debra likes to play Seven Schemers, Pirates in Java Island, Dwarves and Swords.\
+Walter is connected to John, Levi, Bryant.\
+Walter likes to play Seahorse Adventures, Ninja Hamsters, Super Mushroom Man.\
+Levi is connected to Ollie, John, Walter.\
+Levi likes to play The Legend of Corgi, Seven Schemers, City Comptroller: The Fiscal Dilemma.\
+Ollie is connected to Mercedes, Freda, Bryant.\
+Ollie likes to play Call of Arms, Dwarves and Swords, The Movie: The Game.\
+Jennie is connected to Levi, John, Freda, Robin.\
+Jennie likes to play Super Mushroom Man, Dinosaur Diner, Call of Arms.\
+Robin is connected to Ollie.\
+Robin likes to play Call of Arms, Dwarves and Swords.\
+Freda is connected to Olive, John, Debra.\
+Freda likes to play Starfleet Commander, Ninja Hamsters, Seahorse Adventures."
+
+# ----------------------------------------------------------------------------- 
+# create_data_structure(string_input): 
+#   Parses a block of text (such as the one above) and stores relevant 
+#   information into a data structure. You are free to choose and design any 
+#   data structure you would like to use to manage the information.
+# 
+# Arguments: 
+#   string_input: block of text containing the network information
+#
+#   You may assume that for all the test cases we will use, you will be given the 
+#   connections and games liked for all users listed on the right-hand side of an
+#   'is connected to' statement. For example, we will not use the string 
+#   "A is connected to B.A likes to play X, Y, Z.C is connected to A.C likes to play X."
+#   as a test case for create_data_structure because the string does not 
+#   list B's connections or liked games.
+#   
+#   The procedure should be able to handle an empty string (the string '') as input, in
+#   which case it should return a network with no users.
+# 
+# Return:
+#   The newly created network data structure
+def create_data_structure(s):
+    network = {} # {user1:[[connections],[games]], user2:[[connections],[games]]...} 
+    user = []
+    connection = []
+    game = []
+    string_ls = s.split('.')
+    #print string_ls
+    if len(s) > 1:
+        
+        for e in string_ls:
+        
+            if 'is connected to' in e:
+                s = e[e.find('is connected to') + 16:] #s is the names of the
+                #connection and the next string to be used
+                #print s
+                s = s.split(', ')
+                user_name = e[:e.find(' is connected to')]
+                user.append(user_name)
+                connection.append(s)
+                #print connection, user
+            
+            if 'likes to play' in e:
+                s = e[e.find('likes to play') + 14:] # "+13" will leave a space
+                #before the game title, e.g. " The Legend of Corgi".
+                s = s.split(', ')
+                game.append(s)
+                #print game
+            else:
+                #return []
+                pass
```

Comment:
since you're not doing anything here, you actually don't need the `else` block at all